src/pages/Nav/FindProduct.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useState } from 'react';
+import { useEffect } from 'react';
+
+const SearchId = ({ searchInput }) => {
+  const [feedData, setFeedData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/productlist.json')
+      .then(response => response.json())
+      .then(result => setFeedData(result));
+  }, []);
+
+  const searchIdData = feedData.filter(e => e.name.includes(searchInput));
+
+  if (searchInput.length > 0 && searchIdData.length > 0) {
```

Comment:
```suggestion
  if (searchInput. && searchIdData) {
```
searchInput이 빈 문자열이라면 falsy한 값을 가져서 false처럼 사용됩니다!
아래 공식문서를 참고하세요

https://developer.mozilla.org/ko/docs/Glossary/Falsy

Reply:
감사합니다!!

Reply:
(searchInput.length > 1 && searchIdData.length > 1)
 2번째 글까지 검색하면 나오게 하고싶은데 그때도 위의 코드 말고 다른 방법이 있을까요??